views/layout.html

```diff
@@ -0,0 +1,124 @@
+{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework. -#}
```

Comment:
comment references extensions - not sure what this means so remove it?

Reply:
Will do. It was there in the prototype code.

Reply:
Didn't want to remove the comment completely - sounds like it would mean something to someone - so just generalised it.